school-manager-server/src/main/resources/application.yml

```diff
@@ -0,0 +1,19 @@
+server:
+  port: 8080
+spring:
+  datasource:
+    url: jdbc:mysql://${MYSQL_HOST:localhost}:3306/HW2
+    username: root
+    password: 1qazXSW@3edc
+    driver-class-name: com.mysql.cj.jdbc.Driver
+  jpa:
+    show-sql: true
+    properties:
+      hibernate:
+        format_sql: true
+        jdbc:
+          batch_size: 10000
+        order_inserts: true
+    hibernate:
+      #create-drop - Dev | update - Prod
+      ddl-auto: create-drop
```

Comment:
1) For prod you will be do nothing in this case. All database modification will be done by flyway. 
2) You should have different profiles if you want to separate environments. (application-dev.yaml)